trillian_admin_api.proto

```diff
@@ -20,7 +20,7 @@ option java_outer_classname = "TrillianApiProto";
 
 package trillian;
 
-import "trillian.proto";
+import "github.com/google/trillian/trillian.proto";
```

Comment:
The problem is that protoc can't distinguish 
`trillian.proto` with `-I=.` from
`github.com/google/trillian/trillian.proto` with `-I=$GOPATH/src`

The current scheme of using local paths eg `trillian.proto` works great as long as these proto files don't get referenced from other proto files outside this directory. But as soon as we have  a personality that wants to proxy some of these types (eg. #578), we'll have a problem:
- The personality imports `github.com/google/trillian/trillian.proto`
- The personality imports `github.com/google/trillian/trillian_log_api.proto`
- which imports `trillian.proto`, which is interpreted as a third file rather than the same as the first, causing issues.